setup.py

```diff
@@ -14,10 +14,8 @@
            ],
       },
       install_requires=[
-           'buildapi-client>=0.1'
            'mozillapulse',
            'mozci>=0.25.5',
-           'requests',
```

Comment:
For services like trigger-bot and pulse_actions what I actually do is a list every simple dependency and all related dependencies and I fix the version of all of them (==). This makes it easier to keep a production system under further control since sometimes bad dependencies can be released into the cheese shop and our production system becomes unstable (obviously this could be mitigated with full proper testing but that's another story). Since we will eventually move to pulse_actions it will make this fine.
I brought all this up since I learned this in the last year and I thought it would be good knowledge to share.